src/main/php/PDepend/Metrics/Analyzer/CrapIndexAnalyzer.php

```diff
@@ -246,6 +246,9 @@ private function createCoverageReport()
     {
         $factory = new Factory();
 
-        return $factory->create($this->options['coverage-report']);
+        /** @var string */
+        $coverageReport = $this->options['coverage-report'];
```

Comment:
I realize it should use the constant:
```suggestion
        $coverageReport = $this->options[self::REPORT_OPTION];
```